src/common/dto/transaction-base-dto.ts

```diff
@@ -17,4 +17,13 @@ export class TransactionBaseDto {
   @IsString()
   @IsUrl()
   readonly webhookUrl?: string;
+
+  @ApiProperty({
+    description:
+      'An optional property that when set to `true` will will verify the validity of the transaction without submitting it to the chain',
+    example: true,
+  })
+  @IsBoolean()
+  @IsOptional()
+  readonly dryRun?: boolean;
```

Comment:
```suggestion
  @ApiPropertyOptional({
    description:
      'Indicator to verify the validity of the transaction without submitting it to the chain',
    type: 'boolean',
    example: false,
  })
  @IsBoolean()
  @IsOptional()
  readonly dryRun?: boolean;
```
I would keep the example value as false, so that everytime someone is using swagger doesn't have to set it to false